### ISSUE TITLE
Something missing on SIP page

### ISSUE THREAD

**Issue by lornajane** [Nexmo/nexmo-developer#1272]
## Description

From Javier: something missing on https://developer.nexmo.com/voice/sip/configure/freeswitch (after it says: `Add the following to sip_profiles/internal.xml:`)

## What did you expect to happen?

See screenshots in the config.

```
I think that there is a screenshot missing.
We have :`Modify autoload_configs/acl.conf.xml and allow traffic from Nexmo's IPs`

----Screenshot about whitelisting IP addresses---

`Add the following to sip_profiles/internal.xml`:
`Create a public dial plan for Nexmo in dialplan/public/nexmo_sip.xml`:

---Screenshot about dial plan----
```

## What actually happened?

No screenshots

## Related resources

https://developer.nexmo.com/voice/sip/configure/freeswitch

**Comment by mheap** [comment]
It looks as though this information never existed, even when the content was migrated from the old docs site. I'm looking for help internally to populate it

**Comment by mheap** [comment]
Resolved by @giannico-vonage via #1295 🙌
